drivers/metal/rendering_device_driver_metal.mm

```diff
@@ -1040,7 +1040,11 @@ static const API_AVAILABLE(macos(11.0), ios(14.0)) MTLSamplerBorderColor SAMPLER
 	{
 		t.serialize(p_writer)
 	} -> std::same_as<void>;
+
+	// Need to disable clang-format as it will remove the mandatory semi-colon here.
+	// clang-format off
 };
+// clang-format on
```

Comment:
This is the sole "hack" needed to make the whole PR compiling.

Reply:
Removed the hack. Found out that the `concept` keyword is not Objective-C++, but C++20. I fixed the formatting by setting the `Standard` rule to `c++20`.